src/index.ts

```diff
@@ -47,82 +38,141 @@ async function runTest(files: string[], options: Options) {
     } = options;
     let coverageEnabled = coverageReporters.length > 0;
 
-    let rocaFiles = [
-        "tap.brs",
-        "roca_lib.brs",
-        "roca_main.brs",
-        "assert_lib.brs",
-    ].map((basename) => path.join(__dirname, "..", "resources", basename));
-
-    let reporterStream = new TapMochaReporter(reporter);
-    let allTestFiles = [...rocaFiles];
+    // Get the list of files that we should load into the execution scope.
+    // Loading them here ensures that they only get lexed/parsed once.
+    let rocaFiles = ["roca_lib.brs", "assert_lib.brs"].map((basename) =>
+        path.join(__dirname, "..", "resources", basename)
+    );
+    let inScopeFiles = [...rocaFiles];
```

Comment:
Looks like `rocaFiles` is only used once!  Would it make sense to inline it in `inScopeFiles`'s initializer?

```suggestion
    let inScopeFiles = ["roca_lib.brs", "assert_lib.brs"].map((basename) =>
        path.join(__dirname, "..", "resources", basename)
    );
```